CHANGES.rst

```diff
@@ -92,6 +92,9 @@ API Changes
   - ``.fts`` and ``.fts.gz`` files will be automatically identified as
     ``io.fits`` files if no explicit ``format`` is given. [#5211]
 
+  - ``delay_doc_updates`` contextmanager to postpone the formatting of the
```

Comment:
I'd say it is a new feature, but I'm not sure we really want to expose it as such, so my feeling would be in "Others" section only.